spec/rubocop/cop/style/extra_spacing_spec.rb

```diff
@@ -247,7 +247,8 @@
     end
 
     it 'does not register an offense if assignments are aligned' do
-      inspect_source(cop, ['a   = 1',
+      inspect_source(cop, ['# comment',
```

Comment:
I actually created a fix for this same issue, but stumbled a bit to figure out what the exact issue was, but I'm pretty sure [this guard clause](https://github.com/akihiro17/rubocop/blob/fix-equal-sign-alignment/lib/rubocop/cop/style/extra_spacing.rb#L57) is involved. (Adding a new-line to the end of the `#inspect_source` argument made the test function correctly.) It would be great if the updated test could reflect what's going on here.